docs/reference/field_types.rst

```diff
@@ -108,6 +108,36 @@ Parameter                               Description
         ;
     }
 
+``TemplateRegistry::TYPE_BOOLEAN``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use the following parameters:
```

Comment:
Maybe we should the term "options" instead of "parameters", in order to be consistent with the wording of the form types.

Reply:
I agree. I replace all occurences.